tests/vse_sync_pp/test_muxed.py

```diff
@@ -0,0 +1,82 @@
+### SPDX-License-Identifier: GPL-2.0-or-later
+import json
+from collections import namedtuple
+from decimal import Decimal
+from io import StringIO
+from itertools import zip_longest
+from unittest import TestCase
+
+from vse_sync_pp.parsers import PARSERS
+from vse_sync_pp.source import muxed
+
+"""Test cases for vse_sync_pp.source.muxed"""
+
+CaseValue = namedtuple("CaseValue", "input,expected")
+NO_PARSER = CaseValue(
+    {
+        "id": "im-not-a-parser",
+        "data": ["I", "should", "not", "be"],
+    },
+    None,
+)
+DPLL_LIST = CaseValue(
+    {
+        "id": "dpll/time-error",
+        "data": ["1876878.28", "3", "3", "-0.79"],
+    },
+    ("dpll/time-error", (Decimal("1876878.28"), 3, 3, Decimal("-0.79"))),
+)
+DPLL_DICT = CaseValue(
+    {
+        "id": "dpll/time-error",
+        "data": {
+            "timestamp": "1876878.28",
+            "eecstate": "3",
+            "state": "3",
+            "terror": "-0.79",
+        },
+    },
+    ("dpll/time-error", (Decimal("1876878.28"), 3, 3, Decimal("-0.79"))),
+)
+GNSS_LIST = CaseValue(
+    {
+        "id": "gnss/time-error",
+        "data": ["681011.839", "5", "-3"],
```

Comment:
```suggestion
        "data": ["681011.839", "3", "5", "-3"],
```

Reply:
Is this to test that it can take what the collector is producing currently?